.storybook/themes/news.scss

```diff
@@ -1367,6 +1367,40 @@
 					margin: 0,
 					padding: 0,
 				),
+				medium-promo: (
+					components: (
+						attribution: (
+							display: block,
+							margin-top: var(--global-spacing-4),
+						),
+						media-item: (
+							float: right,
+							max-width: 100px,
+						),
+						heading: (
+							font-size: var(--global-font-size-7),
+							font-weight: var(--global-font-weight-7),
+							line-height: var(--global-line-height-4),
+							margin-bottom: var(--global-spacing-4),
+						),
+						paragraph: (
+							display: inline,
+							font-size: var(--global-font-size-4),
```

Comment:
This should be using the body alias tokens for font and line height

Reply:
Updated. In fact, these two properties were not needed.